submissions/kkkovalllchuk/html-css-popup/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>POP up</title>
+    <link rel="icon" href="/pop up/img/favicon.png" type="image/png" />
+    <link rel="stylesheet" href="styles.css">
+    
+</head>
+<body>
+    <nav class="nav">
+        <ul class="nav__lists menu">
+            <li class="nav__list-item ">
+                <a href="#" class="nav__list-link" target="_blank"><img src="/pop up/img/dot_checklist_list_todo_icon_226578.svg"alt=""></a>
+            </li>
+            <li class="nav__list-item">
+                <a href="#" class="nav__list-link "
+                    target="_blank"><img src="/pop up/img/home_mobile_ui_button_icon_226573.svg" alt=""></a>
+            </li>
+            <li class="nav__list-item">
+                <div class="menu__popup main">
+                    <input type="checkbox" id="popup" class="popup-input input" />
+                    <label for="popup" class="popup-label"></label>
+                    <div class="popup-main">
+                        <a href="#" class="popup-item" target="_blank">
+                            <img src="/pop up/img/icons8-facebook-новый.svg" alt="facebook" />
+                            <p>Facebook</p>                         
+                        </a>
+                        <a href="#" class="popup-item" target="_blank">
+                            <img src="/pop up/img/icons8-pinterest.svg" alt="pinterest" />
+                            <p>Pinterest</p>
+                        </a>
+                        <a href="#" class="popup-item" target="_blank">
+                            <img src="/pop up/img/icons8-линкедин.svg" alt="Linkedn" />
+                            <p>Linkedn</p>
+                        </a>
+                        <a href="#" class="popup-item" target="_blank">
+                            <img src="/pop up/img/icons8-реддит.svg" alt="reddit" />
+                            <p>Reddit</p>
+                        </a>
+                        <a href="#" class="popup-item hidden" target="_blank">
+                            <img src="/pop up/img/icons8-твиттер.svg" alt="twitter" />
+                            <p>Twitter</p>
+                        </a>
+                        <input type="checkbox" id="more-btn" class="more-input input" />
```

Comment:
Watch [the video](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) to find out how to hide the checkbox in the right way.